java/ql/integration-tests/all-platforms/java/buildless-sibling-projects/test.py

```diff
@@ -0,0 +1,49 @@
+from create_database_utils import *
+from diagnostics_test_utils import *
+from buildless_test_utils import *
+
+import tempfile
+import os.path
+import sys
+
+#The version of gradle used doesn't work on java 17
```

Comment:
Why not switch to a newer version of gradle ?

Reply:
It needs doing, for all the gradle tests in this repo; it seems to involve more than just bumping the version though so I need to learn to forward-convert Gradle to do this